presto-tests/src/test/java/com/facebook/presto/tests/TestDistributedQueriesNoHashGeneration.java

```diff
@@ -15,13 +15,14 @@
 
 import com.google.common.collect.ImmutableMap;
 
+import static com.facebook.presto.tests.FeatureSelection.Feature.ENGINE_ONLY;
 import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunner;
 
 public class TestDistributedQueriesNoHashGeneration
         extends AbstractTestQueries
 {
     public TestDistributedQueriesNoHashGeneration()
     {
-        super(() -> createQueryRunner(ImmutableMap.of(), ImmutableMap.of("optimizer.optimize-hash-generation", "false")));
+        super(() -> createQueryRunner(ImmutableMap.of(), ImmutableMap.of("optimizer.optimize-hash-generation", "false")), FeatureSelection.all().without(ENGINE_ONLY).build());
```

Comment:
maybe we could make shortcut for: `FeatureSelection.all().without(ENGINE_ONLY).build()`?

Reply:
No longer needed.